server/data/educationAndWorkPlanClient.ts

```diff
@@ -33,6 +35,12 @@ export default class EducationAndWorkPlanClient {
     })
   }
 
+  async getGoalsByStatus(prisonNumber: string, status: GoalStatusValue, token: string): Promise<GetGoalsResponse> {
+    return EducationAndWorkPlanClient.restClient(token).get<GetGoalsResponse>({
+      path: `/action-plans/${prisonNumber}/goals?status=${status}`,
+    })
```

Comment:
Is it worth using the `query` property, rather than concatenating the string? We do it in a couple of other places like this:
```
  async getLearnerEducationPage(prisonNumber: string, token: string, page = 0): Promise<LearnerEductionPagedResponse> {
    return CuriousClient.restClient(token).get<LearnerEductionPagedResponse>({
      path: `/learnerEducation/${prisonNumber}`,
      query: {
        page,
      },
    })
```

Reply:
Done